test/spec/ol/style/icon.test.js

```diff
@@ -1,5 +1,5 @@
 import {getUid} from '../../../../src/ol/index.js';
-import {iconImageCache} from '../../../../src/ol/style.js';
+import {shared as iconImageCache} from '../../../../src/ol/style/IconImageCache';
```

Comment:
We import with the `.js` Extension elsewhere, sobI think it should be done the same way here.

Reply:
I added one remark. Please merge when addressed. Thanks for the effort on this!

Reply:
Good catch.  Addressed in e28305186c139545d366fe99bf44e1c7fa5d047f.